suricata.yaml.in

```diff
@@ -168,6 +168,19 @@ outputs:
             # payload-length: yes      # enable dumping payload length, including the gaps
             # packet: yes              # enable dumping of packet (without stream segments)
             # metadata: no             # enable inclusion of app layer metadata with alert. Default yes
+            # If you want metadata, use:
+            # metadata:
```

Comment:
this looks suspicious: is metadata a bool or an object?

Reply:
`metadata` is both -- it's a bit odd, imo